### ISSUE TITLE
javac code.Symbol.complete NullPointerException

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#8989]
https://ci.eclipse.org/openj9/job/Test_openjdk14_j9_extended.system_x86-64_windows_Nightly/46
SharedClasses.SCM01.SingleCL_0
```
CSC 26/03/20 00:43:28: Compiling java files
CSC Compilation failed
CSC stderr An exception has occurred in the compiler (14-internal). Please file a bug against the Java compiler via the Java bug reporting page (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) for duplicates. Include your program, the following diagnostic, and the parameters passed to the Java compiler in your report. Thank you.
CSC stderr java.lang.NullPointerException
CSC stderr 	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:670)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.code.Symbol$PackageSymbol.members(Symbol.java:1164)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.resolveImports(TypeEnter.java:356)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.runPhase(TypeEnter.java:324)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.doCompleteEnvs(TypeEnter.java:285)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:254)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter.complete(TypeEnter.java:201)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:670)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1383)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.Enter.complete(Enter.java:584)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.comp.Enter.main(Enter.java:561)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterTrees(JavaCompiler.java:1071)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:936)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.lambda$doCall$0(JavacTaskImpl.java:104)
CSC stderr 	at com.sun.tools.javac.api.JavacTaskImpl$$Lambda$41/0000000000000000.call(Unknown Source)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.handleExceptions(JavacTaskImpl.java:147)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:100)
CSC stderr 	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:94)
CSC stderr 	at net.openj9.test.sc.JavaGen.compileJavas(JavaGen.java:376)
CSC stderr 	at net.openj9.test.sc.JavaGen.go(JavaGen.java:102)
CSC stderr 	at net.openj9.test.sc.JavaGen.main(JavaGen.java:94)
STF 00:43:39.757 - **FAILED** Process CSC ended with exit code (1) and not the expected exit code/s (0)
```

All  subsequent SCM tests failed
`**FAILED** at step 1 (Copy sharedClasses jar). Expected return value=1 Actual=0 at C:/Users/jenkins/workspace/Test_openjdk14_j9_extended.system_x86-64_windows_Nightly/openjdk-tests/\TKG\test_output_15851931988770\SharedClasses.SCM01.MultiCL_0/20200326-004342-SharedClasses/setUp.pl line 51.`

SharedClasses.SCM01.MultiCL_0 
SharedClasses.SCM01.MultiThread_0
SharedClasses.SCM01.MultiThreadMultiCL_0
SharedClasses.SCM23.SingleCL_0
SharedClasses.SCM23.MultiCL_0
SharedClasses.SCM23.MultiThread_0
SharedClasses.SCM23.MultiThreadMultiCL_0

Similarly 
https://ci.eclipse.org/openj9/job/Test_openjdk14_j9_extended.system_ppc64le_linux_Nightly/47
https://ci.eclipse.org/openj9/job/Test_openjdk14_j9_extended.system_x86-64_linux_Nightly/47
https://ci.eclipse.org/openj9/job/Test_openjdk14_j9_extended.system_ppc64_aix_Nightly/28

https://ci.eclipse.org/openj9/job/Test_openjdk14_j9_sanity.system_x86-64_linux_Nightly/48
```
01:46:41  build-java:
01:46:41      [javac] Compiling 71 source files to /home/jenkins/workspace/Test_openjdk14_j9_sanity.system_x86-64_linux_Nightly/openjdk-tests/system/openjdk-systemtest/openjdk.test.math/bin
01:46:49      [javac] An exception has occurred in the compiler (14-internal). Please file a bug against the Java compiler via the Java bug reporting page (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) for duplicates. Include your program, the following diagnostic, and the parameters passed to the Java compiler in your report. Thank you.
01:46:49      [javac] java.lang.NullPointerException
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:670)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.completeOwners(ClassFinder.java:317)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.complete(ClassFinder.java:290)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.ClassFinder$$Lambda$46/0000000000000000.complete(Unknown Source)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:670)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1383)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.flags(Symbol.java:1304)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$12.visitClassType(Types.java:2133)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$12.visitClassType(Types.java:2116)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Type$ClassType.accept(Type.java:1010)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$DefaultTypeVisitor.visit(Types.java:4853)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.asSuper(Types.java:2113)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$12.visitClassType(Types.java:2129)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$12.visitClassType(Types.java:2116)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Type$ClassType.accept(Type.java:1010)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$DefaultTypeVisitor.visit(Types.java:4853)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.asSuper(Types.java:2113)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$4.visitClassType(Types.java:1179)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$4.visitClassType(Types.java:1100)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Type$ClassType.accept(Type.java:1010)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types$DefaultTypeVisitor.visit(Types.java:4853)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.isSubtype(Types.java:1096)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.isSubtypeUncheckedInternal(Types.java:1022)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.isSubtypeUnchecked(Types.java:1008)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.isConvertible(Types.java:607)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.code.Types.isAssignable(Types.java:2318)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Check$1.compatible(Check.java:556)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Check.checkType(Check.java:597)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Check.checkType(Check.java:584)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.attributeAnnotationValues(Annotate.java:476)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.attributeAnnotation(Annotate.java:432)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.annotateNow(Annotate.java:341)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.lambda$annotateLater$0(Annotate.java:267)
01:46:49      [javac] 	at com.sun.tools.javac.comp.Annotate$$Lambda$184/0000000000000000.run(Unknown Source)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.flush(Annotate.java:191)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.unblockAnnotations(Annotate.java:144)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.comp.Annotate.enterDone(Annotate.java:157)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterDone(JavaCompiler.java:1752)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterTrees(JavaCompiler.java:1073)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:936)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:316)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:176)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.Main.compile(Main.java:57)
01:46:49      [javac] 	at jdk.compiler/com.sun.tools.javac.Main.main(Main.java:43)

```

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.openjdk_s390x_linux_Nightly/12/
java/security/PermissionCollection/PermissionCollectionStreamTest.java.PermissionCollectionStreamTest (from java_security_PermissionCollection_PermissionCollectionStreamTest.java)
```
java.lang.NullPointerException
	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.completeOwners(ClassFinder.java:313)
	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.complete(ClassFinder.java:289)
	at jdk.compiler/com.sun.tools.javac.code.ClassFinder$$Lambda$41/0000000000000000.complete(Unknown Source)
	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.members(Symbol.java:1264)
	at jdk.compiler/com.sun.tools.javac.jvm.ClassReader$AnnotationDeproxy.findAccessMethod(ClassReader.java:2135)
	at jdk.compiler/com.sun.tools.javac.jvm.ClassReader$AnnotationDeproxy.deproxyCompound(ClassReader.java:2126)
	at jdk.compiler/com.sun.tools.javac.jvm.ClassReader$AnnotationDeproxy.deproxyCompoundList(ClassReader.java:2115)
	at jdk.compiler/com.sun.tools.javac.jvm.ClassReader$AnnotationCompleter.run(ClassReader.java:2334)
	at jdk.compiler/com.sun.tools.javac.comp.Annotate.flush(Annotate.java:191)
	at jdk.compiler/com.sun.tools.javac.comp.Annotate.unblockAnnotations(Annotate.java:144)
	at jdk.compiler/com.sun.tools.javac.comp.Annotate.enterDone(Annotate.java:157)
	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterDone(JavaCompiler.java:1741)
	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterTrees(JavaCompiler.java:1068)
	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:937)
	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:311)
	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:170)
	at jdk.compiler/com.sun.tools.javac.Main.compile(Main.java:75)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.javatest.regtest.agent.RegressionCompileCommand.run(RegressionCompileCommand.java:189)
	at com.sun.javatest.regtest.agent.CompileActionHelper.runCompile(CompileActionHelper.java:92)
	at com.sun.javatest.regtest.agent.AgentServer.doCompile(AgentServer.java:220)
	at com.sun.javatest.regtest.agent.AgentServer.run(AgentServer.java:190)
	at com.sun.javatest.regtest.agent.AgentServer.main(AgentServer.java:65)

```

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.openjdk_x86-64_linux_Nightly/13
java/lang/invoke/accessProtectedSuper/Test.java.Test (from java_lang_invoke_accessProtectedSuper_Test.java)
java/lang/invoke/lambda/LambdaReceiver.java.LambdaReceiver (from java_lang_invoke_lambda_LambdaReceiver.java)
ava/lang/invoke/lambda/T8032697.java.T8032697 (from java_lang_invoke_lambda_T8032697.java)
java/security/KeyStore/PKCS12/EntryProtectionTest.java.EntryProtectionTest (from java_security_KeyStore_PKCS12_EntryProtectionTest.java)
java/util/LinkedList/ComodifiedRemove.java.ComodifiedRemove (from java_util_LinkedList_ComodifiedRemove.java)
java/util/concurrent/BlockingQueue/CancelledProducerConsumerLoops.java.CancelledProducerConsumerLoops (from java_util_concurrent_BlockingQueue_CancelledProducerConsumerLoops.java)

last passing
OpenJ9: 18cf44b
OMR: 4175c2c
OpenJDK14: 77c5e42 - openj9-openjdk-jdk14

first failing
OpenJ9: 752c590
OMR: e9b8511
OpenJDK14: 77c5e42 - openj9-openjdk-jdk14

https://github.com/eclipse/openj9/compare/18cf44b...752c590
https://github.com/eclipse/openj9-omr/compare/4175c2c...e9b8511

**Comment by DanHeidinga** [comment]
https://github.com/eclipse/openj9/issues/8988 looks similar as well though it's both an NPE and JIT crash

**Comment by pshipton** [comment]
Seems to be resolved by #8995
